sql/signal_handler.cc

```diff
@@ -58,49 +58,70 @@ extern const char *optimizer_switch_names[];
 
 static inline void output_core_info()
 {
-  /* proc is optional on some BSDs so it can't hurt to look */
-#if defined(HAVE_READLINK) && !defined(__APPLE__) && !defined(__FreeBSD__)
+#ifndef __WIN__
```

Comment:
Better  do not use __WIN__ , it is [deprecated in higher versions](https://github.com/mariadb/server/commit/3d6eb7afcfb4#diff-f07460fa16c227caac9e71893e5e08cf2eea9cba2226677a285f3f37361dac3dR30). Use _WIN32 instead.

Reply:
Ack. I thought you cleaned all these up. Sorry some have crept back in and I foolishly copied them. Thanks for the reminder.